### ISSUE TITLE
[PagePartBundle] Container parameter in PagePartAdmin can be null

### ISSUE THREAD

**Issue by dannyvw** [Kunstmaan/KunstmaanBundlesCMS#3431]
The container parameter in the constructor can be null and then this line will trigger an error https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/7.x/src/Kunstmaan/PagePartBundle/PagePartAdmin/PagePartAdmin.php#L270

Maybe we can remove the container and inject only the event dispatcher?

**Comment by acrobat** [comment]
Yes it's a good idea to inject the correct service but the container property is protected so we need to add a bc layer so this property can be removed in 8.0 and we can already inject the new dependency.
